### ISSUE TITLE
Insufficient text display and customization options for accessibility

### ISSUE THREAD

**Issue by techczech** [typora/typora-issues#2106]
The font size options in the settings is really not sufficient to cover the basic accessibility needs. I had to go and change the CSS to make the display suit my needs. The theming power is great but normal users need better control within the app. The themes should just be presets with more advanced options.

These are the settings that would make sense in the app:
* font size
* line spacing
* line length (in characters ) - could be margins but it's better expressed in how many chars per line
* text colour
* text background

In fact, the absence of these is what's stopped me from recommending Typora more widely.
